src/nwb_conversion_tools/datainterfaces/ecephys/basesortingextractorinterface.py

```diff
@@ -99,26 +101,37 @@ def subset_sorting(self):
 
     def run_conversion(
         self,
-        nwbfile: NWBFile = None,
-        metadata: dict = None,
-        stub_test: bool = False,
-        save_path: OptionalFilePathType = None,
+        nwbfile_path: OptionalFilePathType = None,
+        nwbfile: Optional[NWBFile] = None,
+        metadata: Optional[dict] = None,
         overwrite: bool = False,
+        stub_test: bool = False,
         write_ecephys_metadata: bool = False,
+        save_path: OptionalFilePathType = None,  # TODO: to be removed
```

Comment:
Maybe adding the return to docstring?
Is the only thing I could find.

Reply:
Looks like that's missing from all the others, too - might as well do them all in one-shot in a follow up